flumine/backtest/simulated.py

```diff
@@ -119,9 +125,14 @@ def place(
     def _create_place_response(
         self, bet_id: int, status: str = "SUCCESS", error_code: str = None
     ) -> SimulatedPlaceResponse:
+        order_status = "EXECUTABLE"
```

Comment:
Would it be cleaner to add `order_status` as a var and pass it? As I believe you know when it has expired when called

Reply:
I've addressed this with 
https://github.com/jsphon/flumine/commit/ef75cac7c2d3c7579574a498aa49ca753fdd8ae2